optuna/study/study.py

```diff
@@ -1323,6 +1323,10 @@ def copy_study(
     The direction(s) of the objective(s) in the study, trials, user attributes and system
     attributes are copied.
 
+    .. note::
+        :func:`optuna.copy_study` copies a study even if the optimization is working on.
```

Comment:
I'm not sure but I think the notation that omits the path is more commonly used in Optuna.
```suggestion
        :func:`~optuna.copy_study` copies a study even if the optimization is working on.
```

Reply:
Thank you, I applied. And I found there are both styles in `study.py`. So, I'll open another PR to fix other place e.g. https://github.com/optuna/optuna/blob/15e9f38e4a448f2c1775e46e396ba7fc190df537/optuna/study/study.py#L1305.